service/qos/service_level_controller.cc

```diff
@@ -122,7 +123,14 @@ future<> service_level_controller::update_service_levels_from_distributed_data()
 
     return with_semaphore(_global_controller_db->notifications_serializer, 1, [this] () {
         return async([this] () {
-            service_levels_info service_levels = _sl_data_accessor->get_service_levels().get0();
+            service_levels_info service_levels;
+            try {
+                service_levels = _sl_data_accessor->get_service_levels().get0();
+            } catch (...) {
+                sl_logger.warn("update_service_levels_from_distributed_data: an error occurred"
```

Comment:
The commit message is confusing about the log levels. You said "Most logs are added in the info level since they are not expected to happen frequently" but if they are *not* frequent, how does this mean we should make them info level? Also until I found this warn() I wasn't sure there will be any (yes, the commit message says "most" - but what about the rest?)